arangodb-net-standard.Test/CollectionApi/CollectionApiClientTest.cs

```diff
@@ -217,5 +217,26 @@ public async Task GetCollectionCountAsync_ShouldSucceed()
             }));
             Assert.Equal(HttpStatusCode.NotFound, exception.ApiError.Code);
         }
+
+        [Fact]
+        public async Task RenameCollectionAsync_ShouldSucceed()
```

Comment:
Other test cases we could cover:

- collection name not found
- attempting to rename to an invalid collection name